src/libsyncengine/update_detection/file_system_observer/computefsoperationworker.cpp

```diff
@@ -29,15 +29,15 @@ namespace KDC {
 
 ComputeFSOperationWorker::ComputeFSOperationWorker(std::shared_ptr<SyncPal> syncPal, const std::string &name,
                                                    const std::string &shortName) :
-    ISyncWorker(syncPal, name, shortName),
-    _syncDb(syncPal->_syncDb), _localSnapshot(syncPal->_localSnapshot), _remoteSnapshot(syncPal->_remoteSnapshot) {}
+    ISyncWorker(syncPal, name, shortName), _syncDb(syncPal->_syncDb), _localSnapshot(syncPal->_localSnapshot),
```

Comment:
Would it possible to use public accessors instead of private members?

Reply:
Done in 6c66776ee849ef7120694f5f3064e21c10907145